### PR TITLE
Adapt to Dashboard replacement

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
It adapts RH Che to dashboard replacement that will be done in 7.26 https://github.com/eclipse/che/pull/18549

### CODE REVIEW

File: dockerfiles/che-fabric8/assets/branding/product.json [hunk 1]
```diff
@@ -5,12 +5,6 @@
     "faq": "https://www.eclipse.org/che/docs/che-7/hosted-che/#hosted-che-faq-and-troubleshooting_hosted-che"
   },
   "configuration": {
-    "menu": {
-      "disabled": ["organizations", "factories"]
-    }, 
-    "features": {
-      "disabled": ["workspaceSharing", "kubernetesNamespaceSelector"]
```
Comment (sleshchenko) [inline dockerfiles/che-fabric8/assets/branding/product.json:12]: @ibuziuk Do you see any issue if new dashboard will show readonly namespace on Create Workspace page?
Screenshot_20210114_165650
Screenshot_20210114_165659

File: dockerfiles/che-fabric8/assets/branding/product.json [hunk 2]
```diff
@@ -5,12 +5,6 @@
...
-    },
     "prefetch": {
       "cheCDN":"/api/cdn-support/paths"
```
Comment (sleshchenko) [inline dockerfiles/che-fabric8/assets/branding/product.json:9]: it's not implemented yet in new dashboard. There is an issue https://github.com/eclipse/che/issues/18728

Comment (sleshchenko) [reply dockerfiles/che-fabric8/assets/branding/product.json:9]: BTW should be even implement it?..
We can implement `prefetch.resources` which is common enough and cheCDN should depends on the plan if it's going to be reused like on che-dev cluster.

File: dockerfiles/che-fabric8/assets/branding/product.json (hunk 1 above)
Comment (ibuziuk) [reply dockerfiles/che-fabric8/assets/branding/product.json:12]: this is fine I believe

File: dockerfiles/che-fabric8/assets/branding/product.json (hunk 2 above)
Comment (ibuziuk) [reply dockerfiles/che-fabric8/assets/branding/product.json:9]: I believe any approach will work as long as pre-fetching will be possible to speed up the time for lodaing. Adding @davidfestal who initially developed the CDN prefetching mechanism

**Comment by ibuziuk** [comment]
[test]

File: dockerfiles/che-fabric8/assets/branding/product.json (hunk 2 above)
Comment (davidfestal) [reply dockerfiles/che-fabric8/assets/branding/product.json:9]: The way it currently works is the following:
- The dashboard was pointing to the `/api/cdn-support/paths` endpoint to retrieve the list of resources to prefetch.
- The `/api/cdn-support/paths` endpoint was implemented by an `rh-che` extension to:
    - extract the CDN index annotation (containing a JSON list of CDN artifacts associated to the given version of Che-Theia) from the che-theia docker image configured as the default editor
    - return it

File: dockerfiles/che-fabric8/Dockerfile [hunk 3]
```diff
@@ -4,11 +4,9 @@
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 ARG CHE_DASHBOARD_VERSION=next
-ARG CHE_DASHBOARD_NEXT_VERSION=next
 ARG CHE_WORKSPACE_LOADER_VERSION=next
```
Comment (olexii4) [inline dockerfiles/che-fabric8/Dockerfile:7]: Do we really need a che-workspace-loader?

**Comment by ibuziuk** [comment]
rebasing the branch on top of https://github.com/redhat-developer/rh-che/pull/2043 (7.26.0-SNAPSHOT)

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
@sleshchenko @akurinnoy currently the new UD is failing to load on Hosted Che:
 
peek

could you please take a look?

**Comment by sleshchenko** [comment]
@ibuziuk it should work after https://github.com/eclipse/che-plugin-registry/pull/820 is pulled

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
@sleshchenko the UD issue has been fixed \o/ verified it manually
are you ok if we merge this PR to `upstream-check-7.26.0-SNAPSHOT` or do you plan some other modifications in the short run?

**Comment by ibuziuk** [comment]
merging to `upstream-check-7.26.0-SNAPSHOT`

**Review by ibuziuk — APPROVED** [review]

**Pull request merged**